### PR TITLE
Adding conda-sphinx-theme and redesign of front page

### PR DESCRIPTION
**Pull request by travishathaway**
### Description

This is a redesign of the conda homepage. Please comment on this pull request with thoughts and ideas for improvement.
 
Fixes: #903
Fixes: #747 
Fixes: #907 
Fixes: https://github.com/conda/conda-docs/issues/745

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,59 +1,137 @@
+===================
+Conda Documentation
+===================
+
+Conda provides package, dependency and environment management for any language.
```
Comment (kathatherine) [inline docs/source/index.rst:5]:
```suggestion
Conda provides package, dependency, and environment management for any language.
```

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,59 +1,137 @@
...
+The following documentation site provides all you need to get started with
+leveraging the power of conda.
+
+Install  :octicon:`download;1em;sd-text-primary`
```
Comment (kathatherine) [inline docs/source/index.rst:9]:
```suggestion
Install :octicon:`download;1em;sd-text-primary`
```
Not sure if that extra space was there on purpose.

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,59 +1,137 @@
...
-   :align: center
-   :width: 50%
+.. grid:: 1 2 2 2
+
+    .. grid-item::
+
+        .. card:: Getting started :octicon:`rocket;1em;sd-text-primary`
+            :link: https://docs.conda.io/projects/conda/en/stable/user-guide/getting-started.html
+
+            Learn how to get started using conda with a tutorials and quick start guides
```
Comment (kathatherine) [inline docs/source/index.rst:93]:
```suggestion
            Learn how to get started using conda with tutorials and quick start guides
```

**Comment by travishathaway** [comment]
@kathatherine,

I made some updates to the landing page. Please review them once more. I added four projects to the projects section. I figure we can just add more later as needed, but please let me know if there any "must haves" I missed for the initial launch.

**Review by kathatherine — APPROVED** [review]
Looks great! I don't feel like there are any other "must-haves" for the initial launch.

However, conda-project also started to use the theme, yes? Does that mean we want to include it in the project list?

**Comment by kathatherine** [comment]
Good point about conda-incubator. Works for me!

**Comment by jaimergp** [comment]
`constructor`? :D

**Review by kenodegard — APPROVED** [review]

**Pull request merged**